### DISCUSSION TITLE
🔗 The New Editor Toolbar - adding the Dimensions at the top (Duplicate of #22432)

### DISCUSSION THREAD

**Discussion by stixen84** [elementor/elementor#24713] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Working with the new Editor Toolbar (which i understand is in Alpha), it is currently missing the Width and Height Dimensions in the top right corner that the old bar has.  This is a really handy feature to have when trying to make sure things are responsive across certain widths. I don't want to have to switch to the frontend every time when checking sizes.

![Screenshot 2024-01-07 at 8 09 43 am](https://github.com/elementor/elementor/assets/13925381/8ec60463-cbeb-4555-b5d0-dd48e6a12ded)

### Describe the solution you'd like.

It would be nice if the team could somehow re-incorporate this into the new editor bar. 

There is room next to the search icon where it would fit, or maybe after the Mobile size Icon would be better.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by hitsgee** [comment]
I personally don't have a need for this and would rather have a more minimal interface. You should know your responsive sizes based on site settings and I would always recommend testing in Chrome tools where you can more accurately test on different devices such as iPhones and Samsung's etc. Just my two cents...

**Comment by joshmitch2000** [comment]
I believe this is a duplicate of [#24404](https://github.com/orgs/elementor/discussions/24404). Feel free to go there and upvote that one.

**Comment by nicholaszein** [comment]
Hello there!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #22432

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
